src/bitwarden_workflow_linter/utils.py

```diff
@@ -113,12 +113,14 @@ class Settings:
     enabled_rules: list[dict[str, str]]
     approved_actions: dict[str, Action]
     actionlint_version: str
+    default_branch: str
 
     def __init__(
         self,
         enabled_rules: Optional[list[dict[str, str]]] = None,
         approved_actions: Optional[dict[str, dict[str, str]]] = None,
         actionlint_version: Optional[str] = None,
+        default_branch: str = "main",
```

Comment:
Default to None, as this should always be set from the value of settings.yaml or default_settings.yaml
```suggestion
        default_branch: str = None,
```